src/JSON.mo

```diff
@@ -23,6 +25,7 @@ module JSON {
 
     public func show(json : JSON) : Text = switch (json) {
         case (#Number(v)) { Int.toText(v); };
+        case (#Float(v)) { Float.format(#fix 2, v); };
```

Comment:
We should not lose precision in the textual representation.

Reply:
I updated the code so it returns the exact format.